src/include/timestamp.h

```diff
@@ -35,6 +35,8 @@ struct __wt_time_window {
     wt_timestamp_t stop_ts;         /* default value: WT_TS_MAX */
     uint64_t stop_txn;              /* default value: WT_TXN_MAX */
 
+    wt_timestamp_t prepare_ts;
+    uint64_t prepared_txn;
```

Comment:
Should this be named as prepared_id?

Reply:
i named it as `prepared_txn` to keep it consistent with `start_txn` and `stop_txn` in this struct

Reply:
done